src/metabase/channel/impl/slack.clj

```diff
@@ -149,6 +150,19 @@
    (format "*%s*\n%s" (:name filter) (shared.params/value-string filter (system/site-locale)))
    attachment-text-length-limit))
 
+(defn- include-branding?
+  "Branding in exports is included only for instances that do not have a whitelabel feature flag."
+  []
+  (not (premium-features/enable-whitelabeling?)))
+
+(def metabase-branding-link
+  "Metabase link with UTM params related to the branding exports campaign"
+  "https://www.metabase.com?utm_source=product&utm_medium=export&utm_campaign=exports_branding&utm_content=slack")
+
+(def metabase-branding-copy
+  "Human visible Markdown content that we use for branding purposes in Slack links"
+  "Made with Metabase :blue_heart:")
+
```

Comment:
we should make these vars private by adding `^:private` after the `def`

Reply:
But I'm referencing them in tests.

Reply:
in test you can use `#'channel.slack/metabase-branding-link` to refer to it, it's a hack so we don't have to make these def public.